main-settings/src/main/scala/sbt/Structure.scala

```diff
@@ -264,27 +336,125 @@ object Scoped {
   }
   implicit def richAnyTaskSeq(in: Seq[AnyInitTask]): RichAnyTaskSeq = new RichAnyTaskSeq(in)
   final class RichAnyTaskSeq(keys: Seq[AnyInitTask]) {
-    def dependOn: Initialize[Task[Unit]] = Initialize.joinAny[Task](keys).apply(deps => nop.dependsOn(deps: _*))
+    def dependOn: Initialize[Task[Unit]] =
+      Initialize.joinAny[Task](keys).apply(deps => nop.dependsOn(deps: _*))
   }
 
   // this is the least painful arrangement I came up with
```

Comment:
The way to exclude this block is by surrounding it with

```
// format: off
```

and then

```
// format: on
```

same for the t2ToApp2, etc block below.

I can do it in a follow up PR if you'd prefer.